src/components/renderer/form-masked-input.vue

```diff
@@ -40,6 +40,8 @@ import ValidationMixin from '@processmaker/vue-form-elements/src/components/mixi
 import DataFormatMixin from '@processmaker/vue-form-elements/src/components/mixins/DataFormat';
 import Inputmasked from './form-input-masked';
 import { TheMask } from 'vue-the-mask';
+import { getUserDateFormat, getUserDateTimeFormat } from '@processmaker/vue-form-elements/src/dateUtils';
```

Comment:
I wonder if it's worth exposing these functions as part of what `@processmaker/vue-form-elements` exports?

Although this doesn't seem cause any issues, babel by default ignores imports from `node_modules`, so these imports won't actually be transpiled by babel, but we're not using any fancy new JavaScript features so there's no issues.